bcolz/toplevel.py

```diff
@@ -273,22 +288,28 @@ def fill(shape, dflt=None, dtype=np.float, **kwargs):
     # Create the container
     expectedlen = kwargs.pop("expectedlen", length)
     if dtype.kind == "V" and dtype.shape == ():
-        raise ValueError("fill does not support ctables objects")
-    obj = bcolz.carray([], dtype=dtype, dflt=dflt, expectedlen=expectedlen,
+        list_carrays = []
+        base_rootdir = kwargs.pop('rootdir', None)
+        if base_rootdir is not None:
+            os.mkdir(base_rootdir)
+        for name,col_dype in dtype.descr:
+            if base_rootdir is not None:
+                kwargs['rootdir'] = "{0}/{1}".format(base_rootdir, name)
```

Comment:
Not sure, but I'm afraid you'r right, anyway, I think I'll remove that. When creating those carrays, I need to create the base dir too; once created in the end I try to create the ctable container, it abiously complains because the directory already exists.

I think I'll create the carrays in memory if that's fine (unless it is a bad idea), for the sake of an easier review I'll make a new commit for that